Buildings/Electrical/AC/ThreePhasesBalanced/Loads/MotorDrive/InductionMotors/Validations/package.mo

```diff
@@ -0,0 +1,10 @@
+within Buildings.Electrical.AC.ThreePhasesBalanced.Loads.MotorDrive.InductionMotors;
+package Validations "Contains Validations of the Induction Motor"
+  extends Modelica.Icons.ExamplesPackage;
+
+  annotation (Documentation(info="<html>

```

Comment:
Could you please add more detailed introduction of the reference data? Basically, you could reuse some wordings in the paper. It would also be good to add the introduction in the two validation models.